superset-frontend/src/components/Chart/chartAction.js

```diff
@@ -248,17 +248,20 @@ export async function getChartDataRequest({
 
 export function runAnnotationQuery({
   annotation,
-  timeout = 60,
+  timeout,
   formData = null,
   key,
   isDashboardRequest = false,
   force = false,
 }) {
   return function (dispatch, getState) {
-    const sliceKey = key || Object.keys(getState().charts)[0];
+    const {charts, common} = getState()
+    const sliceKey = key || Object.keys(charts)[0];
+    const queryTimeout = timeout || common.conf.SUPERSET_WEBSERVER_TIMEOUT;
```

Comment:
This config was already used elsewhere, so I decided to reuse it instead of creating a new one. Maybe in a follow up PR if we feel want the webserver and client timeouts to be different, we can update all of them. But for now, it's consistent with other parts of the js codebase.